src/main/java/makaroshyna/onlinebookstore/service/user/UserServiceImpl.java

```diff
@@ -29,4 +37,15 @@ public UserRegistrationResponseDto register(UserRegistrationRequestDto requestDt
         user.setRoles(roleService.getAllByName(USER));
         return userMapper.toDto(userRepository.save(user));
     }
+
+    @Override
+    public UserLoginResponseDto authorize(UserLoginRequestDto requestDto) {
+        Authentication authentication = authenticationManager.authenticate(
+                new UsernamePasswordAuthenticationToken(
+                        requestDto.getEmail(),
+                        requestDto.getPassword())
+        );
+        String token = jwtUtil.generateToken(authentication.getName());
+        return new UserLoginResponseDto(token);
+    }
```

Comment:
let's move this method to another service `AuthenticationService` according to SRP